website/public/main.css

```diff
@@ -784,6 +784,14 @@ kbd.keystroke {
   }
 }
 
+/* Scrollable container */
+@media screen and (max-width: 670px) {
+  pre, code {
+    overflow-x: scroll;
```

Comment:
Minor: Is there a reason you chose `scroll` instead of `auto`? Don't think we want code blocks to have a scroll bar if the code isn't that wide.

Reply:
Good call, i change it to `auto`.